packages/mlocate/src-locate.c.patch

```diff
@@ -4,6 +4,18 @@ with Android 7.0.
 diff -u -r ../mlocate-0.26/src/locate.c ./src/locate.c
 --- ../mlocate-0.26/src/locate.c	2012-09-22 03:48:54.000000000 +0200
 +++ ./src/locate.c	2017-09-22 20:57:44.062275402 +0200
+@@ -973,7 +973,10 @@
+       if (stat (entry, &st) != 0)
+ 	{
+ 	  if (conf_quiet == false)
+-	    error (0, errno, _("can not stat () `%s'"), entry);
++	    {
++	      error (0, errno, _("can not stat () `%s'"), entry);
```

Comment:
I feel this should better be sent upstream

Reply:
And anyways, if seems like the upstream already has fix on master branch. It would make sense to build from master just like Arch

Reply:
Reverted.

I'd like to let another PR bump version.